tests/e2e/framework/common.go

```diff
@@ -176,6 +176,27 @@ func (f *Framework) PrintROSADebugInfo(t *testing.T) {
 	}
 }
 
+func (f *Framework) setUpProfileBundle(pbName string, baselineImage string, contentFile string) (*compv1alpha1.ProfileBundle, error) {
+	origPb := &compv1alpha1.ProfileBundle{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      pbName,
+			Namespace: f.OperatorNamespace,
+		},
+		Spec: compv1alpha1.ProfileBundleSpec{
+			ContentImage: baselineImage,
+			ContentFile:  contentFile,
+		},
+	}
+	// Pass nil in as the cleanupOptions since so we don't invoke all the
+	// cleanup function code in Create. Use defer to cleanup the
+	// ProfileBundle at the end of the test, instead of at the end of the
+	// suite.
+	if err := f.Client.Create(context.TODO(), origPb, nil); err != nil {
+		return "", err
+	}
```

Comment:
We could wire up the defer here, too:

```go
defer f.Client.Delete(context.TODO(), origPb)
```

Looks like we're doing that each time we setup a profile bundle anyway.

Reply:
I thought of that too, but was uncertain how much it would be deferred.
Is it until we exit the function? Or exit the test?

If it is deferred until end of function, won't it be deleted right after we exit `setUpProfileBundle()?

Reply:
I think I cannot put the `defer` in the function:
```
Defer

A defer statement defers the execution of a function until the surrounding function returns. 
```

Reply:
Ack - thank for checking.